code/clue.js

```diff
@@ -1,59 +1,215 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
 
-const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
-  age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+// Murders
+const cluedoMurders = {
+  mrGreen : {
+    firstName: 'Jacob',
+    lastName: 'Green',
+    color: 'green',
+    description: 'He has a lot of connections',
+    age: 45,
+    image: 'assets/green.png',
+    occupation: 'Entrepreneur'
+  },
+  profPlum : {
+    firstName: 'Victor',
+    lastName: 'Plum',
+    color: 'purple',
+    description: 'He is financially ruined',
+    age: 64,
+    image: 'assets/plum.png',
+    occupation: 'Looking for opportunities'
+  },
+  missScarlet : {
+    firstName: 'Cassandra',
+    lastName: 'Scarlet',
+    color: 'red',
+    description: 'Golddigger',
+    age: '"25"',
+    image: 'assets/scarlet.png',
+    occupation: 'Classified' 
+  }, 
+  mrsPeacock : {
+    firstName: 'Eleonor',
+    lastName: 'Peacock',
+    color: 'blue',
+    description: 'Crazy',
+    age: 51,
+    image: 'assets/peacock.png',
+    occupation: 'Hired gun' 
+  },
+  colMustard : {
+    firstName: 'Jack',
+    lastName: 'Mustard',
+    color: 'yellow',
+    description: 'Solid',
+    age: 83,
+    image: 'assets/mustard.png',
+    occupation: 'No one knows' 
+  }
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
-
-const rope = {
-  name: 'rope',
-  weight: 10
+// Weapons
+const cluedoWeapons = {
+  rope : {
+    name: 'Rope',
+    weight: 2,
+    weightUnit: "kg",
+    length: 20,
+    lengthUnit: "cm"
+  },
+  knife : {
+    name: 'Knife',
+    weight: 1,
+    weightUnit: "kg",
+    length: 30,
+    lengthUnit: "cm"
+  },
+  candlestick : {
+    name: 'Candlestick',
+    weight: 4,
+    weightUnit: "kg",
```

Comment:
Seems like both `weightUnit` and `lengthUnit` is the same everywhere here. Maybe that property could be a more "general" object instead of in everywhere here?